documents/knowledge/elements.yaml

```
@@ -2405,10 +2405,12 @@
   overloads:
       num: ~a
       str: any_upper(a)
-  vectorise: true
+      lst: filter(lst, is_true)
```

```suggestion
      lst: keep truthy
```
I don't think there's much point in writing code if it's not the actual code we use. Just an English phrase should be enough